manager/rbac_manager.py

```diff
@@ -68,17 +68,9 @@ def __eq__(self, other):
 
 class RbacManager(metaclass=Singleton):
     """Class represents RBAC handling and verification."""
-    session = None
-    rbac_url = None
-    disable_rbac = None
-    rbac_timeout = None
-
-    def __init__(self):
-        self.session = requests.Session()
-        if CFG.rbac_url:
-            self.rbac_url = f"{CFG.rbac_url}/api/rbac/v1/access/?application=vulnerability,inventory"
-        self.disable_rbac = CFG.disable_rbac
-        self.rbac_timeout = CFG.rbac_timeout
+
+    session = requests.Session()
```

Comment:
I'd avoid using HTTP sessions, in OpenShift environment it maintains connection to single pod and doesn't utilize all pods equally, making problems with scaling.